src/compiler/comments.ts

```diff
@@ -32,6 +33,7 @@ namespace ts {
             emitNodeWithComments,
             emitBodyWithDetachedComments,
             emitTrailingCommentsOfPosition,
+            emitLeadingComments,

```

Comment:
thanks @rbuckton ! I was wondering if I should create similar function call like `emitTrailingCommentsOfPosition`